snmp/assets/dashboards/troubleshooting_dashboard.json

```diff
@@ -0,0 +1,2267 @@
+{
+    "author_name": "Datadog",
+    "description": null,
+    "layout_type": "ordered",
+    "template_variables": [
```

Comment:
I think it be helpful to add templar vars for:
- device_ip
- exporter_ip (netflow)
- profile
- ...